bin/checkver.ps1

```diff
@@ -1,18 +1,33 @@
-# checks websites for newer versions using an (optional) regular expression defined in the manifest
-# use $dir to specify a manifest directory to check from, otherwise ./bucket is used
+<#
+.SYNOPSIS
+    Check manifest for a newer version.
+.DESCRIPTION
+    Checks websites for newer versions using an (optional) regular expression defined in the manifest.
+.PARAMETER App
+    Manifest name to search.
+    Placeholders are supported.
+.PARAMETER Dir
+    Where to search for manifest(s).
+.PARAMETER Update
+    Update given manifest
+.PARAMETER ForceUpdate
+    Update given manifest(s) even when there is no new version.
+    Useful for hash updates.
+.PARAMETER SkipSupported
+    Updated manifests will not be shown.
+.EXAMPLE
+    PS $BUCKETDIR $ .\bin\checkver.ps1
+    Check all manifest inside default directory.
+#>
 param(
-    [String]$app,
-    [String]$dir,
-    [Switch]$update = $false,
-    [Switch]$forceUpdate = $false
+    [String] $App = '*',
+    [ValidateScript( {Test-Path $_ -Type Container})]
```

Comment:
I like that! Throwing a message instead of a generic error would be better I guess.
```powershell
    [ValidateScript({
        if(!(Test-Path $_ -Type Container)) {
            throw "$_ is not a directory!"
        }
        $true
    })]
```

Reply:
`Cannot validate argument on parameter 'Dir'. C:\COSI\ALFA is not a directory!`
is better than
`Cannot validate argument on parameter 'Dir'.  The "Test-Path $_ -Type Container" validation script for the argument with value "iamnotadirectory.txt" did not
return a result of True. Determine why the validation script failed, and then try the command again.`